coordinator/src/main/scala/filodb.coordinator/queryplanner/SingleClusterPlanner.scala

```diff
@@ -117,14 +119,17 @@ class SingleClusterPlanner(val dataset: Dataset,
                            val queryConfig: QueryConfig,
                            clusterName: String,
                            spreadProvider: SpreadProvider = StaticSpreadProvider(),
-                           targetSchemaProvider: TargetSchemaProvider = StaticTargetSchemaProvider(),
+                           _targetSchemaProvider: TargetSchemaProvider = StaticTargetSchemaProvider(),
```

Comment:
Not too sure what the convention should be-- this is immediately captured by a lambda (`targetSchemaProvider(QueryContext)`) and should never be directly referenced again.